react/src/Header.js

```diff
@@ -56,7 +56,8 @@ const PortalAccessKeyWarning = ({ header }) => {
             </WarningBar>
         } else if (portalAccessKeyInfo.data?.invalid) {
             return <WarningBar>
-                <b>Warning: Access key for associated Portal {portalAccessKeyInfo.data.expired ? 'has expired' : 'is invalid'}</b>
+                <Tooltip id="alert-access-key-invalid" position="bottom" text={`Portal access key: ${portalAccessKeyInfo.data.key}`} />
```

Comment:
This are minor UI updates, if/when create/expiration data not available if access key expired/invalid.

Reply:
Is there some reason you don't want to put this key in the alert itself? As in
```
<b>Alert: <span id="alert-access-key-invalid">Access key {`Portal access key: ${portalAccessKeyInfo.data.key}</span> for associated Portal {portalAccessKeyInfo.data.expired ? 'has expired' : (portalAccessKeyInfo.data.probably_expired ? 'has probably expired' : 'is invalid')}</b>
```
Is it that users shouldn't see it? It is, after all, not the secret key and it is, after all, expired. (I can't see the UI that will result but I have visions of the tooltip obscuring the alert as each tries to supply half of the info.)